templates-and-layout/locked-down-template-inline-bordered.html

```diff
@@ -50,17 +50,32 @@
             // https://www.tiny.cloud/docs/configure/editor-appearance/#fixed_toolbar_container
             fixed_toolbar_container: '#toolbar',
 
+            // Make the toolbar always visible (requires TinyMCE 5.5)
+            // https://www.tiny.cloud/docs/configure/editor-appearance/#toolbar_persist
+            toolbar_persist: true,
+
             setup: (editor) => {
-                // Apply a custom class to the wrapper element when the editor gets focus
-                // https://www.tiny.cloud/docs/advanced/events/
+                // Disable the UI when the editor has loaded
+                // TODO: Use init or PreInit?
```

Comment:
`init` would be when you want to do this, as the UI hasn't been setup at the `PreInit` stage. As Simon mentioned in the other comment, we may even want to use `SkinLoaded`.

Reply:
Now using the skinloaded event